llvm/lib/Transforms/HipStdPar/HipStdPar.cpp

```diff
@@ -37,6 +37,16 @@
 //    memory that ends up in one of the runtime equivalents, since this can
 //    happen if e.g. a library that was compiled without interposition returns
 //    an allocation that can be validly passed to `free`.
+//
+// 3. MathFixup (required): Some accelerators might have an incomplete
```

Comment:
We really need to fix our whole library usage strategy (which I'm working on writing up how)